roles/upgrade/tasks/remove-kernel-modules.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: remove-kernel-modules | Get names of loaded kernel modules
+  ansible.builtin.command:
+    cmd: awk '{print $1}' /proc/modules
+  register: loaded_kernel_modules
+  changed_when: false
+
+- name: remove-kernel-modules | Remove kernel module {{ kernel_module }}
```

Comment:
Variables are not read at the time that task starts, so they are not displayed in the title. This is displayed as plain test `TASK [infra.leapp.upgrade : remove-kernel-modules | Remove kernel module {{ kernel_module }}] ***`. So remove this var.